src/test-utils.ts

```diff
@@ -1,7 +1,24 @@
-import * as msw from 'msw';
-import { setupServer, SetupServerApi } from 'msw/node';
```

Comment:
This is the key, primary change. In order to be able to safely import this file in the React Native environment, we must avoid importing _values_ from `msw/node`.